kairon/data_processor/processor.py

```diff
@@ -904,6 +909,11 @@ def __save_stories(self, story_steps, bot: Text, user: Text):
     def __prepare_training_story_events(self, events, timestamp):
         for event in events:
             if event.type == UserUttered.type_name:
+                entities = [{"start": entity['start'],
```

Comment:
you the pass the entire entites list recieved from the Mongo

Reply:
cannot be passed as it is mongo object while training data requires a dict